config/environments/development.rb

```diff
@@ -46,6 +46,9 @@
   # Highlight code that triggered database queries in logs.
   config.active_record.verbose_query_logs = true
 
+  config.log_level = :info
+  config.logger = Logger.new($stdout)
```

Comment:
Is this a leftoverr change or something for all the devs?

Reply:
I was intending it for all the devs